src/elements/footer-social.html

```diff
@@ -0,0 +1,212 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../bower_components/paper-fab/paper-fab.html">
+
+
+<dom-module id="footer-social">
+  <template>
+    <style is="custom-style" include="shared-styles flex flex-alignment"></style>
+    <style>
+      :host {
+        padding-left: 4px;
+        position: relative;
+        color: var(--footer-text-color);;
+      }
+
+      .title {
+        display: inline-block;
+        text-transform: uppercase;
+        font-weight: 500;
+        font-size: 14px;
+        margin: 0;
+      }
+
+      .nav-inline {
+        display: inline;
+        margin: 0 55px 0 4px;
+      }
+
+      ul.nav-inline {
+        padding-left: 10px;
+      }
+
+      .nav-inline li {
+        display: inline-block;
+      }
+
+      .share-gplus {
+        height: 30px;
+        padding: 8px;
+        width: 23px;
+      }
+
+      .share-gplus paper-icon-button {
+        color: var(--red-gplus-color);
```

Comment:
Just `gplus-color`. Because if they change color we don't need to update variable name.